modules/azure/centos-std/centos-std-startup.sh.tmpl

```diff
@@ -0,0 +1,245 @@
+# Copyright (c) 2019 Teradici Corporation
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+#!/bin/bash
+
+while getopts ":a:b:c:d:e:f:g:h:i:" opt; do
+  case $opt in
+    a) pcoip_registration_code="$OPTARG"
+    ;;
+    b) ad_service_account_password="$OPTARG"
+    ;;
+    c) ad_service_account_username="$OPTARG"
+    ;;
+    d) domain_name="$OPTARG"
+    ;;
+    e) domain_controller_ip="$OPTARG"
+    ;;
+    f) appID="$OPTARG"
+    ;;
+    g) aadClientSecret="$OPTARG"
+    ;;
+    h) tenantID="$OPTARG"
+    ;;
+    i) keyIdentifier="$OPTARG"
+    ;;
+    \?) echo "Invalid option -$OPTARG" >&10
+    ;;
+  esac
+done
+
+INST_LOG_PATH="/var/log/teradici/agent/"
```

Comment:
See if you can break up the startup scripts in separate commits for ease of code reviews.